unittests/deep_mind_tests.cpp

```diff
@@ -100,14 +104,15 @@ BOOST_FIXTURE_TEST_CASE(deep_mind, deep_mind_tester)
       return std::find(argv, argv + argc, std::string("--save-dmlog")) != (argv + argc);
    }();
 
+   auto log_output_path = tmp.file().get_file_path();
+
    if(save_log)
    {
-      // Cannot use fc::copy as it does not copy to an existing destination file 
-      fc::rename(log_output.path().preferred_string(), DEEP_MIND_LOGFILE);
+      std::filesystem::copy(log_output_path, DEEP_MIND_LOGFILE, std::filesystem::copy_options::update_existing);
```

Comment:
This `fc::rename()` became `fs::copy(.. update_existing)`. The semantics are clearly different between the two. Any reason for the change?

Reply:
`log_output_path` is derived from `tmp` of type `temp_cfile object which` would remove the file when `tmp` is out of scope. Simply rename the file would cause the desctructor tryiing to remove an non-existent file.